spec/bolt/cli_spec.rb

```diff
@@ -207,15 +230,6 @@ def stub_config(file_content = {})
         )
       end
 
-      it 'errors if missing a bolt-project.yaml' do
```

Comment:
Not clear why we're removing this, rather than just updating the message?

Reply:
Removed accidentally.

Reply:
Ah, scratch that last comment. Removed them because use of the `module` command requires `modules` be configured, which requires you have a `bolt-project.yaml`. `Bolt::CLI.assert_project_file` is dead code now. 😛